runtime/compiler/runtime/J9Profiler.hpp

```diff
@@ -422,6 +433,32 @@ class TR_ValueProfileInfo
    TR_PERSISTENT_ALLOC(TR_Memory::ValueProfileInfo)
 
    TR_ValueProfileInfo(TR_CallSiteInfo *info);
+
+#if 0
```

Comment:
Can you explain why this is ifdef'd out? Ordinarily we wouldn't accept ifdef'd out code since it is likely to rot and isn't tested or compiled....